rcl_yaml_param_parser/test/test_node_params_descriptors.cpp

```diff
@@ -0,0 +1,84 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+
+#include "../src/impl/node_params_descriptors.h"
+#include "rcutils/allocator.h"
+
+TEST(TestNodeParamsDescriptors, init_fini) {
+  rcutils_allocator_t allocator = rcutils_get_default_allocator();
+  rcl_node_params_descriptors_t node_descriptors = {NULL, NULL, 0u, 128u};
```

Comment:
@bpwilcox here and elsewhere, why are we not zero initializing `rcl_node_params_descriptors_t` structs?